cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -2,6 +2,8 @@
 import fauxfactory
 import pytest
 
+from selenium.common.exceptions import WebDriverException
```

Comment:
What's the need for `WebDriverException` in the test here?  This indicates to me that there is something wrong in `MigrationPlan.wait_for_state` that its raising a `WebDriverException`.

If its a very one-off case of a very specific use of `wait_for_state` that triggers this exception, please import the exception from widgetastic instead of from selenium, to keep the tests decoupled from selenium itself.

```from widgetastic.exceptions import WebDriverException```

Reply:
> What's the need for `WebDriverException` in the test here? This indicates to me that there is something wrong in `MigrationPlan.wait_for_state` that its raising a `WebDriverException`.

Test throwing `WebDriverException` in this case because when we restart evmserverd it also restarts session. This has nothing to do with `wait_for_state`, so even if I add `navigate_to(appliance.server, 'LoggedIn')`  there, it will raise exception while navigation.

> If its a very one-off case of a very specific use of `wait_for_state` that triggers this exception, please import the exception from widgetastic instead of from selenium, to keep the tests decoupled from selenium itself.
> 
> `from widgetastic.exceptions import WebDriverException`

I have updated test to use widgetastic exception instead of selenium